packages/api-sdk/src/medical/client/metriport.ts

```diff
@@ -597,12 +661,63 @@ export class MetriportMedicalApi {
    *
    * @returns True if the signature is verified, false otherwise.
    */
-  verifyWebhookSignature = (wh_key: string, reqBody: string, signature: string): boolean => {
+  verifyWebhookSignature(wh_key: string, reqBody: string, signature: string): boolean {
+    return MetriportMedicalApi.verifyWebhookSignature(wh_key, reqBody, signature);
+  }
+
+  /**
+   * Verifies the signature of a webhook request.
+   * Refer to Metriport's documentation for more details: https://docs.metriport.com/medical-api/more-info/webhooks.
+   *
+   * @param wh_key - your webhook key.
+   * @param req.body - the body of the webhook request.
+   * @param signature - the signature obtained from the webhook request header.
+   *
+   * @returns True if the signature is verified, false otherwise.
+   */
+  static verifyWebhookSignature(wh_key: string, reqBody: string, signature: string): boolean {
     const signatureAsString = String(signature);
     const receivedHash = crypto
       .createHmac("sha256", wh_key)
       .update(JSON.stringify(reqBody))
       .digest("hex");
     return receivedHash === signatureAsString;
-  };
+  }
```

Comment:
Static so users don't need to instantiate a client just to validate WH requests.